crates/napi/src/doc.rs

```diff
@@ -50,6 +52,20 @@ impl Language for FrontEndLanguage {
   }
 }
 
+impl TryFrom<SupportLang> for FrontEndLanguage {
+  type Error = String;
+  fn try_from(s: SupportLang) -> Result<Self, Self::Error> {
+    match s {
+      SupportLang::Html => Ok(FrontEndLanguage::Html),
+      SupportLang::JavaScript => Ok(FrontEndLanguage::JavaScript),
+      SupportLang::Tsx => Ok(FrontEndLanguage::Tsx),
+      SupportLang::Css => Ok(FrontEndLanguage::Css),
+      SupportLang::TypeScript => Ok(FrontEndLanguage::TypeScript),
+      _ => Err(s.to_string())
```

Comment:
Is the error message only containing the language name? We can make it more helpful I think.